assets/scss/layout/_header.scss

```diff
@@ -1 +1,99 @@
-@import "./nav";
\ No newline at end of file
+@import "./nav";
+
+.topbar-wrapper-boxes {
+  height: 100px;
+  margin: 0 auto;
+  display: flex;
+  box-sizing: border-box;
+  border-bottom: 1px solid $black;
+  .inner-wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    .topbar-box {
+      width: 50%;
+      .topbar-box-social-media {
+        display: flex;
+        justify-content: flex-end;
+      }
+      img {
+        height: 23px;
+        width: 23px;
+        margin-left: 5px;
+      }
+    }
+  }
+}
+.logo-bar {
+  .inner-wrapper {
+    display: flex;
+    align-items: center;
+    padding: 75px 0 25px;
+    justify-content: space-between;
+    .logo-bar-image {
+      width: 50%;
+      &:hover{
+        img{
+          transform: scale(.95);
+        }
+      }
+      img {
+        transition: .25s ease all;
+      }
+    }
+  }
+}
+.header-description-wrapper {
+  padding-bottom: 150px;
+  border-bottom: 1px solid $black;
+  .inner-wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    .header-description-box {
+      width: 100%;
+      justify-content: space-between;
+      padding: 50px 0;
+      display: flex;
+      flex-wrap: wrap;
+      .header-description-heading {
+        width: 50%;
+        span{
+          display: block;
+          transition: 0.8s ease all;
+          transform: translateX(0);
```

Comment:
to musi tak byc?
Jak usuniesz to sie animacja zepsuje?

Reply:
Zrobione nic się nie zepsuło.